services/resource_dereference_service.go

```diff
@@ -32,7 +32,7 @@ func (rds ResourceService) DereferenceResourceMetadata(did string, resourceId st
 
 	var context string
 	if contentType == types.DIDJSONLD || contentType == types.JSONLD {
-		context = types.ResolutionSchemaJSONLD
+		context = types.W3IDDIDURL
```

Comment:
If this is to populate the context file, you know `W3IDDIDURL`'s contents lead to a not found page, right?

Can ask Markus if/when that link is meant to go live and whether to keep it as the DID URL dereferencing link in context (correct, but doesn't exist) or keep it to DID Resolution (does exist, but maybe not the correct context).